examples/plugins/inspec-resource-lister/test/helper.rb

```diff
@@ -23,4 +23,3 @@
 require 'minitest/autorun'
 
 # You might want to put some debugging tools here.  We run tests to find bugs, after all.
-require 'byebug'
```

Comment:
Maybe comment out this line?

I'm all for removing it, but if we do maybe the comment above too.

Reply:
I'm +1 on removing all byebug references, and the comment.